src/Silverback.Integration/Messaging/Configuration/WithCallbacksBrokerOptionsBuilderExtensions.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using Microsoft.Extensions.DependencyInjection;
+using Silverback.Util;
+
+namespace Silverback.Messaging.Configuration
+{
+    /// <summary>
+    ///     Adds methods to register <see cref="IBrokerEventsHandler"/>s to the <see cref="IBrokerOptionsBuilder"/>.
+    /// </summary>
+    public static class WithCallbacksBrokerOptionsBuilderExtensions
```

Comment:
The name doesn't reflect its content anymore, it should be like `AddBrokerEventsHandlerBrokerOptionsBuilderExtensions` (what a huge name, tho).

Reply:
Good catch 👍